### ISSUE TITLE
Do hyperparameter tuning

### ISSUE THREAD

**Issue by Mattizza** [Mattizza/FL-project#18]
Tune learning rate, optimizer and transforms.
